packages/dev-scripts/utils/sf-config.js

```diff
@@ -20,6 +20,7 @@ const PACKAGE_DEFAULTS = {
     // Cleaner errors than ts-node through tests
```

Comment:
SHould we make clean-all consistent too? Also, can you fix the `-clean all` I think it should be `sf-clean all`

Reply:
Sure I'll make that change